src/main/java/jenkins/plugins/nodejs/tools/pathresolvers/LatestInstallerPathResolver.java

```diff
@@ -114,10 +114,17 @@ public String resolvePathFor(String version, Platform platform, CPU cpu) {
             arch = "x64";
             break;
         case arm64:
+            if (nodeVersion.compareTo(new NodeJSVersion(15, 8, 1)) >= 0 || nodeVersion.compareTo(new NodeJSVersion(8, 6, 0)) == 0 || nodeVersion.compareTo(new NodeJSVersion(4, 0, 0)) < 0) {
```

Comment:
It's not clear why you ensure the version is lower than 15.8.1 (actually is not released yet) ? Do you expect arm64 will no more supported by nodejs?

Reply:
I removed the upper bound of 15.8.1, so should automatically allow to download newer versions.